### PR TITLE
French translation for the `size-adjust` CSS descriptor reference

### PR DESCRIPTION
**Pull request by audrasjb**
This PR adds a French translation for the `size-adjust` CSS descriptor reference.
See also #1752, #1753, #1754

### CODE REVIEW

File: files/fr/web/css/@font-face/size-adjust/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+<h2 id="syntax">Syntaxe</h2>
+
+<pre class="brush: css">size-adjust: 90%;</pre>
+
+<h3 id="values">Valeurs</h3>
+
+<dl>
+  <dt><code>&lt;percentage&gt;</code></dt>
+  <dd>Une valeur exprimée en {{cssxref("&lt;percentage&gt;", "&lt;pourcentage&gt;")}}, avec une valeur initiale de 100%.</dd>
```
Comment (SphinxKnight) [inline files/fr/web/css/@font-face/size-adjust/index.html:21]: https://github.com/mdn/translated-content/pull/1752/files#r678013156 again ^^

File: files/fr/web/css/@font-face/size-adjust/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+title: size-adjust
+slug: Web/CSS/@font-face/size-adjust
+browser-compat: css.at-rules.font-face.size-adjust
+translation_of: 'Web/CSS/@font-face/size-adjust'
+---
+<p>{{CSSRef}}{{SeeCompatTable}}</p>
+
+<p>Le descripteur CSS <strong><code>size-adjust</code></strong> CSS définit un multiplicateur destiné aux contours des glyphes et au mesures associées à la police. Cela facilite l'harmonisation de l'apparence des polices lorsqu'elles sont rendues avec la même taille.</p>
```
Comment (SphinxKnight) [inline files/fr/web/css/@font-face/size-adjust/index.html:9]: minor typo "au mesures" > "aux mesures"

**Review by SphinxKnight — CHANGES_REQUESTED** [review]

File: files/fr/web/css/@font-face/size-adjust/index.html (hunk 2 above)
Comment (audrasjb) [reply files/fr/web/css/@font-face/size-adjust/index.html]: Fixed in https://github.com/mdn/translated-content/pull/1755/commits/f24626b749ca42e25dad1fdf7c0a02f83b965931

File: files/fr/web/css/@font-face/size-adjust/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/@font-face/size-adjust/index.html]: Yeah, fixed in https://github.com/mdn/translated-content/pull/1755/commits/1ffa2ac870686f9ad39059e3eef0c62fce5453a6

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
f24626b introduced a .DS_Store file :x

**Comment by audrasjb** [comment]
Crap! Fixed in https://github.com/mdn/translated-content/pull/1755/commits/49cb1e8ebeb51ec8e1b55d4fc8cd0cc452edad93

**Review by SphinxKnight — APPROVED** [review]

**Pull request merged**